src/components/network/tabs/Advanced.js

```diff
@@ -71,7 +81,7 @@ export class Advanced extends React.Component {
      */
     _handleOptionChange(list, selectedIndex, init = false) {
         const {actions, chartValues} = this.props
-        let value = list[selectedIndex]
+        const value = list.find(item => item.name === selectedIndex);
```

Comment:
it is veeery bad to compare name with index, isn't it? ;)

Reply:
I mimicked old code to do quick fix, but yeap it is. Will need to be changed with proper variable later.

Reply:
such quick dirty fixes tend to wonder around for lot of time. Adding TODO in code is a minimum, to inform future reader why you had written such a thing ;)

Even better is to fill an self-issue for this.